ivy/backend_handler.py

```diff
@@ -46,9 +48,16 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 _backend_reverse_dict["ivy.functional.backends.torch"] = "torch"
 
 
+_static_backend_dict = dict()
 # Backend Getting/Setting #
 # ----------------------- #
 
+def _load_static_backends():
+    _static_backend_dict["ivy.functional.backends.numpy"] = importlib.import_module("ivy.functional.backends.numpy")
+    _static_backend_dict["ivy.functional.backends.torch"] = importlib.import_module("ivy.functional.backends.torch")
+    _static_backend_dict["ivy.functional.backends.jax"] = importlib.import_module("ivy.functional.backends.jax")
+    _static_backend_dict["ivy.functional.backends.tensorflow"] = importlib.import_module("ivy.functional.backends.tensorflow")
+
 
 def _determine_backend_from_args(args):
```

Comment:
made changes to `determine_backend_from_args` since it wasn't correctly handling `dict/ivy.Containers`.

Reply:
I suppose we refactor this implementation to make it more cleaner and use a single loop for a `container` or a `list ` or a `tuple` (something resembling the `nest` functions), but we can look into that once the previously mentioned issues are fixed given that the logic works well